src/net/java/sip/communicator/plugin/desktoputil/plaf/SearchFieldUI.java

```diff
@@ -177,10 +180,25 @@ protected void customPaintBackground(Graphics g)
                 - searchIcon.getIconHeight()/2;
 
             g2.drawImage(searchIcon.getImage(), 3, dy, null);
+            String str = c.getText();
```

Comment:
Either:
- Keep using `StringUtils.containsLetters` (from Jitsi-Utils) and undo the changes here
- Or simply test for !isAlpha
Just copy/pasting a method from jitsi-utils to here is besides the point.

Please also fix the formatting and don't double check for null/empty (currently lines 186 and 199/200).

Reply:
Copy pasting methods allows us to remove dependency fully and in the same time save the same mention as previously.
But you are the boss ...

Reply:
But yes. I have better solution